getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -161,6 +161,20 @@ located in the project root:
     </Project>
 
 
-Then using the dotnet command line to restore package, in the project root::
+Whenever packages are added or modified, run nuget restore in the root of the
+project directory, to ensure that the nuget packages will be available for msbuild
+(the tool used by godot to build mono projects) to use, run::
```

Comment:
I think the description in parenthesis is not needed.

Reply:
sure, can remove.